### ISSUE TITLE
Arglist of methods with eql specializers loses package information (keyword in particular)

### ISSUE THREAD

**Issue by zellerin** [melisgl/mgl-pax#32]
As the title says, the generated documentation of eql specializers might look misleading.

Example (tested with the quicklisp version):
```
(in-package  cl-user)

(mgl-pax:defsection @show-kw-loses-colon  ()
  (foo (method () ((eql :bar)))))

(defmethod foo ((par (eql :bar)))
  "XXX"
  par)

(mgl-pax:document @show-kw-loses-colon)
```

generates 
`- [method] FOO (PAR (EQL BAR))`.

I assume that package information in general might be too much information, but the fact that the argument is expected to be a keyword, not symbol from related package, seems to be worthy keeping.

**Comment by melisgl** [comment]
Thank you for the report. Reproduced on the latest version.

**Comment by melisgl** [comment]
Fixed in 473e201d.

**Comment by zellerin** [comment]
Thanks, both for the fix and for the library.
